ADAM/src/AppBundle/Controller/AdvertController.php

```diff
@@ -23,7 +24,7 @@ class AdvertController extends Controller
      * Lists all Advert entities.
      *
      * @Route("/", name="annonce_index")
-     * @Method("GET")
+     * @Template()
```

Comment:
Tu peux supprimer le `@Template()` car ici tu envoies déjà le bon template en faisant ton `return $this->render('AppBundle:adret:index.html.twig' ...`